drivers/can/can_stm32.c

```diff
@@ -372,11 +372,14 @@ int can_stm32_set_timing(const struct device *dev,
 	}
 
 	can->BTR &= ~(CAN_BTR_BRP_Msk | CAN_BTR_TS1_Msk |
-		      CAN_BTR_TS2_Msk | CAN_BTR_SJW_Msk);
-
-	can->BTR |= (((timing->phase_seg1 - 1) & 0x0F) << CAN_BTR_TS1_Pos) |
-		    (((timing->phase_seg2 - 1) & 0x07) << CAN_BTR_TS2_Pos) |
-		    (((timing->sjw        - 1) & 0x07) << CAN_BTR_SJW_Pos);
+		      CAN_BTR_TS2_Msk | CAN_BTR_SJW_Msk |
+		      CAN_BTR_BRP_Msk);
```

Comment:
CAN_BTR_BRP_Msk is doubled now.

Reply:
Oops, stupid me. I suggested to add it to above statement, but seems like it was already there... Sorry for the confusion @jdascenzio.

Reply:
Indeed, I changed quickly without thinking too much ... I removed the duplicate CAN_BTR_BRP_Msk